packages/admin/admin/src/dataGrid/usePersistentColumnState.tsx

```diff
@@ -55,10 +56,13 @@ type GridProps = Omit<DataGridProps, "rows" | "columns"> & {
 export function usePersistentColumnState(stateKey: string): GridProps {
     const apiRef = useGridApiRef();
     const columns = useGridColumns(apiRef);
+    const match = useRouteMatch();
+
+    const storageKeyPrefix = `${btoa(match.path)}${stateKey}`;
```

Comment:
```suggestion
    const storageKeyPrefix = `${match.path}${stateKey}`;
```

I don't think we should encode this, as it makes it harder to debug the local storage